src/components/shared/ButtonWithPanel/ArrowPanel.css

```diff
@@ -2,36 +2,48 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-.arrowPanelAnchor {
-  position: absolute;
-  z-index: 10;
-  top: 75%;
-  left: 50%;
-}
-
+/**
+ * .arrowPanel is a wrapper which renders nothing by itself and
+ * just establishes the bounds of the panel. The actual panel appearance is
+ * rendered by .arrowPanelInnerWrapper.
+ * The bottom edge of .arrowPanel is always at the bottom of the viewport.
+ * This allows .arrowPanelInnerWrapper to grow as needed and then to stop
+ * growing (and become scrollable) once it hits the bottom.
+ *
+ * The following properties are set by inline style: left, top, width
+ * The following CSS variables may be set by users: --arrow-panel-background, --arrow-panel-padding
+ */
 .arrowPanel {
-  --internal-offset-from-right: var(--offset-from-right, 60px);
-  --internal-offset-from-top: 15px;
-  --internal-width: var(--width, initial);
-  --internal-button-height: 30px;
+  --internal-arrow-clip-height: 15px;
+  --internal-background: var(--arrow-panel-background, hsl(0deg 0% 97%));
+  --internal-padding: var(--arrow-panel-padding, 16px);
 
-  position: absolute;
-  top: var(--internal-offset-from-top);
-  right: calc(var(--internal-offset-from-right) * -1);
-  min-width: var(--internal-width);
+  position: fixed;
+  z-index: 2;
+  bottom: 5px; /* leave a small gap at the bottom of the viewport */
+  display: flex;
+  flex-flow: column nowrap;
+  padding-top: var(--internal-arrow-clip-height);
+  margin-bottom: auto;
+  color: black;
+  line-height: 1.3;
+  pointer-events: none;
+  text-align: left;
```

Comment:
It's not clear to me why there's a text-align here?
If you decide to keep it, what do you think about using `start` instead?

Reply:
I don't remember why I put a text-align there. I'll remove it.